src/routes/userapp/components/detail-page-specific/useAddComment.tsx

```diff
@@ -1,19 +1,21 @@
 import { useMutation } from "react-query";
 import { useParams } from "react-router-dom";
 import { BACKEND_URL, queryClient } from "../../../../query";
-import { Comment } from "../../../../types";
+import { NewComment } from "../../types";
+import { getAccessToken } from "../../../../auth/utils";
 
-const addComment = (storeId: string, newComment: Comment) => {
-  // no such path in mock
+const addComment = (itemId: string, newComment: NewComment) => {
+  const token = getAccessToken();
   return fetch(
     `${
       process.env.NODE_ENV === "development"
-        ? `/api/addcomment/${storeId}`
-        : `${BACKEND_URL}/addcomment/${storeId}`
+        ? `/api/items/${itemId}/comments` // nonexisting path
```

Comment:
Nie musisz tutaj sprawdzac czy DEV, bo to juz sprawdzamy w BACKEND_URL. Czyli wystarczy po prostu:

${BACKEND_URL}/items/${itemId}/comments

bez żadnego ternary operatora